tests/acceptance/pageObjects/publicLinkPasswordPage.js

```diff
@@ -0,0 +1,33 @@
+module.exports = {
+  url: function () {
+    return this.api.launchUrl
+  },
+  elements: {
+    body: 'body',
+    passwordInput: {
+      selector: 'input[type=password]'
+    },
+    passwordSubmitButton: {
+      selector: '.oc-login-authorize-button'
+    }
+  },
+  commands: [
+    {
+      /**
+       *
+       * @param {string} password
+       */
+      submitPublicLinkPassword: async function (password) {
+        await this
+          .waitForElementVisible('@passwordInput')
+          .setValue('@passwordInput', password)
+          .click('@passwordSubmitButton')
+
+        return this
+          .page.FilesPageElement.filesList()
+          .waitForElementPresent({ selector: '@filesListProgressBar', abortOnFailure: false }) // don't fail if we are too late
+          .waitForElementNotPresent('@filesListProgressBar')
```

Comment:
with #2082 merged you can now use
```
      return navigationHelper.navigateAndWaitTillLoaded(
        this.url(), this.page.FilesPageElement.filesList().elements.filesListProgressBar.selector
      )
```
see https://github.com/owncloud/phoenix/pull/2082/commits/9f7cac7241f57be5091d63d0e90b999975fa0708